docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -737,6 +737,17 @@ quarkus.http.auth.permission.authenticated.applies-to=JAXRS <1>
 ----
 <1> Tell Quarkus to run the HTTP permission check after the tenant has been selected with the `@Tenant` annotation.
 
+=== Resolve with configuration
```

Comment:
If we agree to run this check after `TenantResolver` then please add this section after the `TenantResolver` section, and also add a link in the same order at https://quarkus.io/guides/security-openid-connect-multitenancy#static-tenant-resolution